docs/specs/organization/brainstorm.md

```diff
@@ -333,5 +331,7 @@ If we wanted to support this, here are some potential problems we need to consid
    Since the ID token has the active organization stored, to switch organization, the user MUST sign in again.
    Require the user to sign in again eliminate a bunch of problems, like how to handle different password policies, different MFA requirements.
 
-4. The idea of computing the most strict password policies / MFA requirements, and then determine which a sign-in is needed it also very to document.
+4. The idea of computing the most strict password policies / MFA requirements, and then determine which a sign-in is needed it also very hard to document.
    If it is hard to document, then it is probably that the developer will have a hard time using it.
+
+5. GitHub allows the user to be member of multiple organizations. During sign-in, the user is not prompted to select an organization. If one of the organization the user belongs to require 2FA, then the user is required to have 2FA. Even Auth0 cannot model this use-case without resorting to Auth0 post-login actions.
```

Comment:
Allow session to be created without organization. This behavior is controlled by a query.